README.md

```diff
@@ -13,7 +13,7 @@ to `puppetserver` to switch to the JVM-based Puppet Server.
 
 Many puppet.conf options for agents, masters and other are parameterized, with
 class documentation provided at the top of the manifests.
-`server_additional_settings` can be used to supply master options that are
+The `*additional_settings` parameters can be used to supply options that are
 missing parameters.
```

Comment:
Maybe add something like 'there is a parameter for each configuration section' or explicitly name them? It is that I know what's going on here, but as a new user I couldn't place this.